sos/collector/clusters/juju.py

```diff
@@ -45,6 +46,7 @@ def __init__(self, model_name):
             self.apps = {}
             self.units = {}
             self.machines = {}
+            self.ui_log = logging.getLogger("sos_ui")
```

Comment:
Isn't it better to use `self.log_warn` directly, to log the message both to console and logfile? (https://github.com/sosreport/sos/blob/main/sos/collector/clusters/__init__.py#L221-L223).

Reply:
If I want to use that, I need to pass the log function into `_get_index`.

Reply:
In that case, I'd say use `getLogger('sos')` instead of `sos_ui`, as this will write to both the file log and to console.